src/k8s-engine/lima.ts

```diff
@@ -1120,11 +1126,9 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     const workdir = await fs.promises.mkdtemp(path.join(os.tmpdir(), 'rd-containerd-install-'));
 
     try {
-      const fixedProfile = path.join(workdir, 'profile');
-      const profileContents = (await fs.promises.readFile(resources.get('scripts', 'profile'), { encoding: 'utf-8' }));
+      const profilePath = path.join(paths.resources, 'scripts', 'profile');
```

Comment:
Yes, no need to read in the file. At one point I was munging the contents before copying to the VM, but there was no need for that. So no need to read it in.